pkgs/development/python-modules/lmnotify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, oauthlib
+, requests
+, requests_oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "lmnotify";
+  version = "0.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1l0h4yab7ix8psf65iygc1wy5xwq3v2rwwjixvd8rwk46d2477dx";
+  };
+
+  propagatedBuildInputs = [ oauthlib requests requests_oauthlib ];
+
+  doCheck = false; # no tests exist
+
+  pythonImportsCheck = [ "lmnotify" ];
+
+  meta = with lib; {
+    description = "lmnotify is a package for sending notifications to LaMetric Time.";
```

Comment:
Descriptions shouldn't start with the package name or end with a period: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
Please fix that in a follow-up PR.